packages/cli/lib/services/sdk.ts

```diff
@@ -170,6 +172,8 @@ export class NangoSyncCLI extends NangoSyncBase {
     releaseLock = NangoActionCLI['prototype']['releaseLock'];
     releaseAllLocks = NangoActionCLI['prototype']['releaseAllLocks'];
 
+    protected showLoggerLevelWarning = createLoggerLevelWarning();
```

Comment:
can it be `showLoggerLeverWarning = NangoActionCLI['prototype']['showLoggerLevelWarning']` like above